app/celery/contact_information_tasks.py

```diff
@@ -76,3 +208,54 @@ def lookup_contact_info(
 
     notification.to = recipient
     dao_update_notification(notification)
+
+    if is_feature_enabled(FeatureFlag.VA_PROFILE_V3_COMBINE_CONTACT_INFO_AND_PERMISSIONS_LOOKUP):
+        if not communication_allowed:
+            current_app.logger.info(
+                'Permission denied for recipient %s for notification %s',
+                va_profile_id,
+                notification_id,
+            )
+            if not permission_message is None:
+                reason = permission_message
+            else:
+                reason = 'Contact preferences set to false'
+            handle_lack_of_permission(notification_id, notification, reason)
+
+
+def get_communication_item_id_for_permission_check(notification_id: str, communication_item_id: str) -> Optional[str]:
```

Comment:
consider a trailing comma here to break these args out onto new lines.

Reply:
Trailing comma added